DependencyInjection/HttplugExtension.php

```diff
@@ -27,6 +27,13 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.xml');
         $loader->load('plugins.xml');
         $loader->load('discovery.xml');
+        if ($config['toolbar']['enabled'] && $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
> It's always a good idea to inject logger. Fingers Crossed handler candecide then when and what to log.

Yes, I meant more of a DataCollector to collect data for the profiler. That is injected all the time